src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -593,11 +597,16 @@ public void selectDropdownByActualValue(WebElement element, String value) {
     }
 
     /**
-     * Returns the status message in the page. Returns "" if there is no
-     *         status message in the page.
+     * Returns a list of texts of user status messages in the page.
```

Comment:
I think it would be phrased better with `a list containing the texts of the user ...`

Reply:
Yes, I guess that sounds better.